app/scripts/views/mixins/settings-panel-mixin.js

```diff
@@ -41,6 +41,7 @@ define(function (require, exports, module) {
       this.closeAllPanels();
 
       this.$('.settings-unit').addClass('open');
+      $('#fxa-settings-content').addClass('animate-shadow');
```

Comment:
can we use `this.$` instead of just `$`. Shane is trying to get rid of those for the WebPack build